docs(revamp): Fix main and multi-cloud README.md, revamp site (#1489)
microsoft/retina: docs/02-Installation/03-Config.md

# Retina Config

> Retina config allow to define the level of granularity and filtering of Retina metrics, as well as additional options related to retina-agent and retina-operator. We can update Retina config to enable captures, annotaations, and define data aggregation, as a few examples.

Reviewer: Should this not be under the "Overview" heading?
If not, do we need the "Overview" heading at all?

> Retina config allow to define the level of granularity and filtering of Retina metrics, as well as additional options related to retina-agent and retina-operator. We can update Retina config to enable captures, annotaations, and define data aggregation, as a few examples.

Reviewer 2: ```suggestion
Retina config allows you to define the level of granularity and filtering of Retina metrics, as well as additional options related to retina-agent and retina-operator. For example, we can update Retina config to enable captures, annotations, and define data aggregation.
```
